Documentation/Concepts/Backend/PageModule/Index.rst

```diff
@@ -1,72 +1,99 @@
+:navigation-title: Page module
 .. include:: /Includes.rst.txt
 .. _page-content:
 
-============
-Pages module
-============
+==============================
+"Page" backend module in TYPO3
+==============================
 
-Site content can be structured in many ways.
+The :guilabel:`Web > Page` module is used by the editors of the site to add
+and modify content elements on the page.
```

Comment:
```suggestion
and modify content elements on the page. A content element 
in TYPO3 is a bundle of pre defined TCA (Table Configuration Array) 
backend fields that we can 
add into our HTML code that is displayed on our website. A 
TCA backend field can be added to a content element. Every time 
a TCA field is filled out and saved, this value is 
stored in the database against this field and content element. And 
every time you add this content element in the backend you can 
fill out the same bundle of fields in the backend. The goal is to 
display the whole website with content elements, except the header 
and footer. 
```

Reply:
Can we move that text of yours to its own concept page or into the glossary? It is out of scope here.